src/components/v5/common/ActionSidebar/partials/forms/ManagePermissionsForm/consts.tsx

```diff
@@ -29,7 +29,7 @@ export const validationSchema = object()
   .shape({
     member: string().required(),
     team: number().required(),
-    createdIn: number(),
+    createdIn: number().defined(),
```

Comment:
I think this change is preventing the form from being submitted when using the permissions decision method.

![Screenshot 2024-02-22 at 10 45 44](https://github.com/JoinColony/colonyCDapp/assets/34915414/65d62b6e-a34c-4ce1-8331-d180e5d0a004)